tools/scripts/icu-file-utf8-check.py

```diff
@@ -116,6 +124,15 @@ def main(argv):
             if check_file(f, source_file) != 0:
                 exit_status = 1
 
+            # Lastly, check the line endings of the file.
+            # Not that 'grep' returns null if it reports a file,
```

Comment:
Not --> Note

Reply:
"... superflous import ..." refers to a quick second commit (stashed) after I realized the script still has 'import subprocess' from some debug work ...

Reply:
Done